template-parts/content.php

```diff
@@ -10,16 +10,25 @@
 
 ?>
 
-<div id="post-<?php the_ID(); ?>" <?php post_class('blogpost-entry'); ?>>
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 	<header>
-		<h2><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></h2>
+		<h1 class="entry-title"><?php the_title(); ?></h1>
```

Comment:
This is going to result in multiple h1 elements on archive pages, which we don't want. There needs to be conditionals throughout this file that use `is_single()` to dictate different content depending on if it's a single post or an archive page.

See here for an example:
https://github.com/WordPress/twentyseventeen/blob/master/components/post/content.php